modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -130,6 +144,18 @@ public void deInit() {
      * Activates a lease updater to renew leases.
      */
     public void activate() {
+        leaseConciliator = new LeaseConciliator(clusterService);
```

Comment:
Busy locks are missing everywhere except stop!

Reply:
Activation happens in buzy lock now.